### PR TITLE
Include instalation methods for quick start

### PR DESCRIPTION
**Pull request by lucasdu4rte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,24 @@ I have the time in milliseconds and I want it to become "30 minutes" or "3 days,
 
 **This library is actively maintained but no new features will be added.**
 
+
+## Installation
```
Comment (EvanHahn) [inline README.md]: Nitpick: could you make sure this heading matches the others?

```diff
-## Installation
+Installation
+------------
```

File: README.md [hunk 2]
```diff
@@ -8,10 +8,24 @@ I have the time in milliseconds and I want it to become "30 minutes" or "3 days,
...
+
+This package is available as *humanize-duration* on [npm](https://www.npmjs.com/package/humanize-duration) and Bower. You can also include the JavaScript file in the browser.
+
+```bash
```
Comment (EvanHahn) [inline README.md]: Nitpick: this isn't necessarily Bash, it's any shell command. Could you change this to `sh` instead?

File: README.md [hunk 3]
```diff
@@ -8,10 +8,24 @@ I have the time in milliseconds and I want it to become "30 minutes" or "3 days,
...
+npm install humanize-duration
+```
+
+Or using `yarn`
```
Comment (EvanHahn) [inline README.md]: I'm personally opposed to Yarn (long story) and wouldn't document it in my readme. Could you remove?

File: README.md (hunk 2 above)
Comment (lucasdu4rte) [reply README.md]: I will change

File: README.md (hunk 3 above)
Comment (lucasdu4rte) [reply README.md]: No problem bro, I will remove

File: README.md (hunk 1 above)
Comment (lucasdu4rte) [reply README.md]: Sorry, I forgot this. Solved now!

**Comment by EvanHahn** [comment]
Thanks!

**Pull request merged**